### PR TITLE
[10.0] [ADD] analytic_wip_report

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Work In Progress Statement
==========================
The Work In Progress Statement is associated to the Percentage of Completion Method, and is used to compile the information necessary for the percentage of completion calculations, but also to provide crucial information about the total value and progress of work on hand inventory.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

File: analytic_wip_report/views/account_analytic_account_view.xml [hunk 1]
```diff
@@ -10,49 +9,49 @@
...
+                <field name="earned_revenue"/>
+                <field name="over_billings"/>
+                <field name="under_billings"/>
+                <field name="state" invisible="1"/>
+                <field name="currency_id" groups="base.group_multi_currency"/>
+                <field name="date" invisible="1"/>
+                <field name="user_id" invisible="1"/>
+                <field name="partner_id" invisible="1"/>
+                <field name="parent_id" invisible="1"/>
+                <field name="account_class"/>
```
Comment (AaronHForgeFlow) [inline analytic_wip_report/views/account_analytic_account_view.xml]: Can you set this field as invisible? It's not needed in the report

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM but one thing

File: analytic_wip_report/README.rst [hunk 2]
```diff
@@ -81,6 +81,7 @@ Contributors
 
 * Eficent <http://www.eficent.com>
 * Matmoz d.o.o. <http://www.matmoz.si>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```
Comment (aheficent) [inline analytic_wip_report/README.rst]: IMHO it should be:
* Jordi Ballester <jordi.ballester@eficent.com>
* Matjaž Mozetič <m.mozetic@matmoz.si>
* Aaron Henriquez <ahenriquez@eficent.com>
* Nikul Chaudhary<nikul.chaudhary.serpentcs@gmail.com>

but it's not big deal as it was like that before

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**